envs/example/defaults-2.0.yml

```diff
@@ -207,6 +207,9 @@ nova:
       disk: 160
       ephemeral: 0
       vcpus: 8
+  service:
```

Comment:
almost every code we write must be enabled for rhosp unless there is strong reason not to do it.